test/cmdlineTests/yul_stack_opt/output

```diff
@@ -5,11 +5,11 @@ Pretty printed source:
 object "object" {
     code {
         {
-            let a1, b1, c1, d1, e1, f1, g1, h1, i1, j1, k1, l1, m1, n1, o1, p1 := fun()
```

Comment:
This is not really fulfulling the objective of the test anymore :(
The idea was to generate many local variables that are unused right after assignment.

Reply:
I'll fix it by making it run a sequence without `p`. Not sure if there is another way around it. I can check if assigning values of `a3, ..., p3` will still make `stack_opt` work.